### PR TITLE
Feature/schema

### PR DESCRIPTION
**Pull request by andela-ssunday**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -3,4 +3,5 @@ var app = express();
 
 app.listen(5000,function(){
 	console.log("Server is up and running.");
```
Comment (andela-aidowu) [inline app.js]: recommended spacing is 2 spaces

File: models/category.model.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+'use strict'
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/andeVote');
```
Comment (andela-aidowu) [inline models/category.model.js]: remove this here and move to your app.js or whatever the name is called

File: models/votes.model.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+'use strict'
+require("./users.model");
+require("./category.model");
+var mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/andeVote');
+
+var Schema = mongoose.Schema;
+var votesSchema = new Schema({
+	user: { type: mongoose.Schema.Types.ObjectId, ref: 'Users' },
```
Comment (andela-aidowu) [inline models/votes.model.js]: indent this properly

File: spec/schemaSpec.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+			}, function(err,res){
+				category.find({}, function(err,res){
+					expect(res).toEqual([]);				
+				});
+		});
+			done();
+		})
+
+
+		// it("should be able to create categories", function(done){
```
Comment (andela-aidowu) [inline spec/schemaSpec.js]: delete line 47 to 70 if you not using it

**Comment by andela-bfowotade** [comment]
You repeated this in three separate files
+mongoose.connect('mongodb://localhost/andeVote'); 
Can you put it in a file and require it? Keeps things much neater.

**Pull request merged**